packages/muelu/src/Graph/UncoupledAggregation/MueLu_AggregationPhase2aAlgorithm_def.hpp

```diff
@@ -114,15 +149,30 @@ namespace MueLu {
             //       with all the other check too big aggregates would not be accepted at all.
             if (aggSize < as<size_t>(maxNodesPerAggregate))
               aggList[aggSize++] = neigh;
+            num_nonaggd_neighbors++;
           }
 
           numNeighbors++;
         }
       }
 
       // NOTE: ML uses a hardcoded value 3 instead of MinNodesPerAggregate
```

Comment:
Is this comment still valid?  If not, please delete.  If yes, then then why use `minNodesPerAggregate`?